cli/src/action/api/rbac.rs

```diff
@@ -43,6 +43,68 @@ impl fmt::Display for Role {
     }
 }
 
+/// Constructs roles for submission to a splinter node.
```

Comment:
Typo in commit message 'changes' -> 'change'

Reply:
Updated.